packages/packages/structure/src/hooks/use-action-props.tsx

```diff
@@ -1,4 +1,3 @@
-import * as React from 'react';
```

Comment:
You can also change it to ts instead of tsx
but it's not related to this PR, so maybe extract this and the things before to another PR?